crates/holochain_state/tests/cache_tests/mod.rs

```diff
@@ -65,7 +63,7 @@ async fn check_state(
 
 #[tokio::test(flavor = "multi_thread")]
 async fn cache_inits_correctly() {
-    let db = test_in_mem_db(DbKindDht(Arc::new(DnaHash::from_raw_32(vec![0; 32]))));
+    let db = test_dht_db_with_id(0).to_db();
```

Comment:
This is a test helper that was created after this test, it's a little easier to look at so thought I'd swap it out